bedrock/firefox/views.py

```diff
@@ -552,6 +552,10 @@ def new(request):
                 template = 'firefox/new/batm/nimble.html'
             elif experience == 'batmresist':
                 template = 'firefox/new/batm/resist.html'
+            elif experience == 'batma':
```

Comment:
Oh yeah, tests. I really must learn how to write those some day. I can at least copy-paste.